### PR TITLE
Re-implement the Graphviz plugin

### PR DESCRIPTION
**Pull request by da-liii**

### CODE REVIEW

File: docs/001.tm [hunk 1]
```diff
@@ -19,7 +20,21 @@
 
   <section|How to test it?>
 
+  <\shell-code>
```
Comment (da-liii) [inline docs/001.tm:23]: only work for macOS and GNU/Linux

For Windows, we have not support it yet. It should be resolved in issue 005.tm.

**Comment by woutersj** [comment]
Thanks for this very nice work!

Can I ask why you want to use `pants` and `pex`? I'm not familiar with them.

**Comment by da-liii** [comment]
Because `pants` is the best build tool for Python now!

And PEX will help us bundle the dependencies. I will re-write the sympy plugin to show how it works!

**Comment by da-liii** [comment]
I merged it because it is still in early stage, and I guess reviewers are not familiar with Pants. And I need to move to rewrite the Python and the SymPy plugin to show how PEX works!

**Comment by da-liii** [comment]
Here is my work on the new SymPy plugin: 
https://github.com/texmacs/plugins-in-python/pull/6

**Comment by woutersj** [comment]
Would the idea be to distribute the `.pex` file with TeXmacs? It would have the advantage of bundling dependencies, but also users would be stuck with the bundled version.

**Comment by da-liii** [comment]
> Would the idea be to distribute the .pex file with TeXmacs?

YES.

**Comment by woutersj** [comment]
I'm not a fan of that approach. Users may want to use different versions than what we bundle. I would use the `pex` file approach as a fallback if Sympy cannot be loaded.

**Comment by da-liii** [comment]
> I would use the pex file approach as a fallback if Sympy cannot be loaded.

A reasonable better solution! I think PEX is important for those who does not know how to manage the Python virtualenv or does not care about the SymPy version.

**Comment by woutersj** [comment]
We could probably use 'system' in the init file to check for modules:

`(if (== 0 (system (string-append (python-command) " -c 'import sympy'"))) '(old-loader) '(pex-loader))`

**Pull request merged**